### PR TITLE
[JBJCA-704] Test for property injection into JDBC connection listeners

### PR DESCRIPTION
**Pull request by mjurc**
Implementing test for property injection in connection listenter test case as per [https://issues.jboss.org/browse/JBJCA-704](https://issues.jboss.org/browse/JBJCA-704).

### CODE REVIEW

File: testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/jca/connectionlistener/AbstractTestsuite.java [hunk 1]
```diff
@@ -31,6 +17,20 @@
...
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.ADD;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OP;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.OP_ADDR;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
+
+
```
Comment (bstansberry) [inline testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/jca/connectionlistener/AbstractTestsuite.java]: The original import ordering was correct.

File: testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/jca/connectionlistener/TestConnectionListener.java [hunk 2]
```diff
@@ -2,37 +2,52 @@
...
 
-import java.sql.Connection;
-import java.sql.SQLException;
-
 import org.jboss.jca.adapters.jdbc.spi.listener.ConnectionListener;
 import org.jboss.logging.Logger;
 
+import java.sql.Connection;
+import java.sql.SQLException;
+
```
Comment (bstansberry) [inline testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/jca/connectionlistener/TestConnectionListener.java]: Same here. Also, please don't reorder imports when no other import changes are made.

**Comment by mjurc** [comment]
I have fixed the erroneous import fixes.

**Comment by ctomc** [comment]
this is ok to test

**Comment by ctomc** [comment]
this is ok to test

**Comment by ctomc** [comment]
this is ok to test

**Comment by ctomc** [comment]
@kabir this is missing downstream PR!

**Comment by ctomc** [comment]
I will include it as part of my checkstyle PR

**Pull request merged**